renovate.json

```diff
@@ -6,6 +6,8 @@
     "schedule": "before 4am"
   },
   "postUpdateOptions": ["gomodTidy", "gomodUpdateImportPaths"],
+  "postUpgradeTasks": { "commands": ["(cd ./actions/installer && make clean && make package)"] },
+  "allowedPostUpgradeCommands": ["(cd ./actions/installer && make clean && make package)"],
```

Comment:
https://docs.renovatebot.com/configuration-options/#postupgradetasks
> "Post-upgrade tasks can only be used on self-hosted Renovate instances."

Unfortunately I don't think we can have rennovate update the `dist/` directory automatically without running a self-hosted instance.

Reply:
Oh I included this by accident while working on something else.
 - https://github.com/slsa-framework/slsa-verifier/pull/754

And yes I found the same to be true.